templates/zerver/emails/digest.html

```diff
@@ -1,32 +1,49 @@
 {% extends "zerver/emails/email_base_messages.html" %}
 
 {% block content %}
-    {% if hot_conversations %}
-        {% for convo in hot_conversations %}
-        <div class='messages'>
-            {% for recipient_block in convo.first_few_messages %}
-                <div class='hot_convo_recipient_block'>
-                    <div class='hot_convo_recipient_header'>{{ recipient_block.header.html|safe }}</div>
-                    <div class='hot_convo_message_content'>
-                        {% for sender_block in recipient_block.senders %}
-                            {% for message_block in sender_block.content %}
-                            <div class='hot_convo_message_content_block'>
-                                {{ message_block.html|safe }}
-                            </div>
+    {% if message_content_disabled_by_realm %}
+        {% if new_messages_count > 0 or new_streams_count > 0 %}
+            <p class="digest_paragraph">
+                {% if new_messages_count > 0 and new_streams_count > 0 %}
+                    {% trans %}You have {{ new_messages_count }} new messages and {{ new_streams_count }} new channels in <a href="{{ realm_url }}">{{ realm_name }}</a>.{% endtrans %}
```

Comment:
Hmmm... you don't really "have" new channels. I think this would read better as "You have N new messages, and there are X new channels in `realm name`." Maybe check on CZO about this though.